### ISSUE TITLE
Issues with new voting dialog

### ISSUE THREAD

**Issue by TheAssassin** [blue-nebula/base#229]
As a long time player, I was very used to the old voting dialog. I've been using the new one for a little over four weeks now, found various issues with it. Here's a (likely incomplete) list.

I miss various features:

- voting for custom maps is not possible
- voting for maps by typing in their name is not possible either
- forcing maps (e.g., as an admin, or on edit servers) so that they're uploaded from a user doesn't seem to be possible

These issues are related to that search field being, well, a search field now. It should probably be a hybrid field: if you enter the name of a map unknown to the game, there must be an option to vote for this anyway.

Some things I find problematic:

- custom server-specified maps are _always_ listed _after_ the bundled ones, i.e., the bundled ones are prioritized, which kills the fun on custom-map servers
- map thumbnails consume a lot of space, increasing the amount of scrolling on long map lists by ~3x
- mutators can't be voted easily, due to the smaller size of these elements
- scrolling with thumbnails enabled is very slow on large lists (really reduces FPS)
- thumbnails are too small to be useful (can hardly recognize them)
- It's hard to vote your map preference (especially if the map is beyond the first two "pages" of items) within the voting time (you need to be fast so others get a chance to vote for your vote, too)
- I don't think that a user can make use of the additional two lines of meta information immediately (can we move these into some sort of popup that is shown when you hover over the entry?)

I think the old preview system was better here, with one large thumbnail preview, and a compact list of maps. It allowed me to vote a _lot_ faster, didn't introduce any significant lag, and was IMO easier to navigate.

I don't want to say "let's introduce the old one again". I consider the overall new UI an improvement. But I think the issues I listed need to be addressed.

**Comment by metalune** [comment]
I agree that these issues need to be addressed. I think putting the extra-information inside a hover-menu would be a great solution (at least for that specific issue)

**Comment by metalune** [comment]
- voting for custom maps is not possible
This should be a fairly simple fix, let's just try to load the map even tho it's not in the list

**Comment by metalune** [comment]
- forcing maps (e.g., as an admin, or on edit servers) so that they're uploaded from a user doesn't seem to be possible
I had recognized this a long time ago and wrote a fix for it. see #162 I kinda lost track of this PR, pardon me.
